src/util/createdb.cpp

```diff
@@ -126,8 +129,11 @@ int createdb(int argc, const char **argv, const Command& command) {
         }
 
         KSeqWrapper* kseq = NULL;
+        std::string seq = ">";
```

Comment:
Can we move this into the if condition?

Reply:
No we can't :( we need to refer to the address of this string later. Variables defined inside `if {}` block is not accessible to the outer scope.